Bonsai.System/IO/GetEnvironmentVariable.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Reactive.Linq;
+
+namespace Bonsai.IO
+{ 
+    /// <summary>
+    /// Represents an operator that returns the value of an OS environment variable.
+    /// </summary>
+    /// 
+    [Description("Returns the value of an environment variable.")]
+    public class GetEnvironmentVariable : Source<string>
+    {
+        [Description("The name of the environment variable to query the value of.")]
+        public string Variable { get; set; }
```

Comment:
This name is consistent with the parameter name of the `GetEnvironmentVariable` method, but I wonder if it would make more sense to just use `Name` for this so we can take advantage of the option to group editing and externalizing of similar properties in the editor.